coops-ui/src/resources/config-objects.ts

```diff
@@ -0,0 +1,68 @@
+
+export class Business {
+     typeEnum: string;
+     displayName:String;
+     flows: Array<Flow>;
+}
+
+export class Flow {
```

Comment:
I know it doesn't matter in JS/TS that this class is declared after it's referenced above, but I still prefer to declare child classes first. Not a big deal though.